<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QViewpointDistributionWidgetBase</class>
 <widget class="QWidget" name="QViewpointDistributionWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>70</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="m_uniformRadioButton">
       <property name="text">
        <string>Uniform viewpoints</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="m_uniformNumberOfViewpointsLabel">
         <property name="text">
          <string>Number of viewpoints</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_uniformNumberOfViewpointsComboBox">
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="m_quasiUniformRadioButton">
       <property name="text">
        <string>Quasi-uniform viewpoints</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="m_quasiUniformRecursionLevelLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Recursion level</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_quasiUniformRecursionLevelSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_quasiUniformNumberOfViewpointsLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>12 viewpoints</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
